dss-api.yml

```diff
@@ -2456,19 +2448,16 @@ definitions:
   CollectionOfCollectionsItem:
     type: object
     properties:
-      collection_uuid:
+      uuid:
         type: string
         description: A UUID identifying the collection.
         pattern: "[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}"
-      collection_version:
-        type: string
+      version:
         description: The version of the UUID identifying the collection.
-      collection:
-        $ref: '#/definitions/Collection'
+        type: string
```

Comment:
What does it mean to drop `dss/storage/identifiers.py`?

Reply:
That object is the whole collection list, like a bundle.  It has unrestricted length so it was decided to not include them.